c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -157,15 +157,21 @@ enum {
     RTR_BLOCKED_BIT = 25,
     RTR_FPE_BIT = 17,
     RTR_LE_BIT = 6,
-    RTR_DENABLE_BIT = 2
+    RTR_PARITY_COUNT_BIT = 5,
+    RTR_FRAME_COUNT_BIT = 4,
+    RTR_TS_COUNT_BIT = 3,
+    RTR_DUMP_ENABLE_BIT = 2
 };
```

Comment:
Some of these bits correspond to the router control register and some to the router status register. Would it make the code clearer (less error prone) if different 'enums' were used for bits in different router registers?

Currently, 'spinnaker.h' contains only register addresses and offsets. Should these bit-level definitions move there to have a common source for hardware-related data?

Reply:
Yes, I agree that things from different registers should be in different places so I have done this.  The masks are still in a single place so far, as these are more the use of the bits so fairly specific to this code.

I would be happy if spinnaker.h contained these sort of definitions since they are hardware specific.  I guess the masks could also be there in that case.  I don't think this directly affects this PR however; it can be done outside of these changes and then a new PR can be done to make more changes to this file.

Reply:
Fully agree.